Source/BaseTableDataDisplayManager/BaseTableDataDisplayManager.swift

```diff
@@ -323,6 +333,38 @@ extension BaseTableDataDisplayManager: UITableViewDelegate {
         self.scrollEvent.invoke(with: guardTable)
     }
 
+    open func tableView(_ tableView: UITableView, canMoveRowAt indexPath: IndexPath) -> Bool {
+        if let generator = cellGenerators[indexPath.section][indexPath.row] as? MovableGenerator {
+            return generator.canMove()
+        }
+        return false
+    }
+
+    open func tableView(_ tableView: UITableView, moveRowAt sourceIndexPath: IndexPath, to destinationIndexPath: IndexPath) {
+        guard sourceIndexPath.section == destinationIndexPath.section else {
+            return
+        }
+        let section = sourceIndexPath.section
+        var oldCellGenerators = cellGenerators[section]
+        let itemToMove = oldCellGenerators[sourceIndexPath.row]
+        oldCellGenerators.remove(at: sourceIndexPath.row)
+        oldCellGenerators.insert(itemToMove, at: destinationIndexPath.row)
+
+        let sectionGenerator = sectionHeaderGenerators[section]
+        removeAllGenerators(in: sectionGenerator)
+        addCellGenerators(oldCellGenerators, toHeader: sectionGenerator)
+
+        cellChangedPosition.invoke(with: (section: section, oldIndex: sourceIndexPath.row, newIndex: destinationIndexPath.row))
```

Comment:
```suggestion
        self.cellChangedPosition.invoke(with: (section: section, 
                                                                         oldIndex: sourceIndexPath.row, 
                                                                         newIndex: destinationIndexPath.row))
```

Reply:
Now there are only two params (was 3), and I think the break to the next line isn't needed